ui/pages/import-token/import-token.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { boolean } from '@storybook/addon-knobs';
+
+import ImportToken from './import-token.component';
+
+export default {
+  title: 'Import Token',
+};
+
+export const ImportTokenComponent = () => {
+  return <ImportToken showSearchTab={boolean('Show Search Tab', false)} />;
```

Comment:
This needs to be `Boolean`

Reply:
When did you change your username, Mark? (cc @Gudahtt, lol)

Reply:
This seems to be the `boolean` function imported from `@storybook/addon-knobs`.  I haven't made any change in the file other than renaming it.

Reply:
ah! oops.